queue/setup.go

```diff
@@ -92,6 +92,10 @@ func (s *Setup) Validate() error {
 		return fmt.Errorf("no queue routes provided")
 	}
 
+	if len(s.PublicKey) == 0 {
```

Comment:
I would add the 32 length check that we see in the pop function: https://github.com/go-vela/server/blob/7ee3ae39552c14a2eb7777195c23a15cd2bf4059/queue/redis/pop.go#L50. Then I would also say that line can be removed once the logic is moved here

Reply:
The s.PublicKey here is the encoded string where c.config.PublicKey is a decoded string. In setup.go, we check for whether the encoded public key is provided upon startup or not.